Tasks/CondaEnvironmentV0/conda.ts

```diff
@@ -7,13 +7,28 @@ import * as internal from './conda_internal';
 import { Platform } from './taskutil';
 
 interface TaskParameters {
-    environmentName: string,
+    customEnvironment?: boolean,
+    environmentName?: string,
     packageSpecs?: string,
     updateConda?: boolean,
+    installOptions?: string,
     createOptions?: string,
     cleanEnvironment?: boolean
 }
 
+// TODO copied from PythonScript. Find a way to share between tasks.
```

Comment:
I'm going to do this in a separate change so I don't have to touch PythonScript in this review.  Maybe this is something we could move to task-lib, but it's mostly useful for the pattern I've been using to write tasks.